<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CAboutDlg</class>
 <widget class="QDialog" name="CAboutDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CAboutDlg</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
	border-image: url(:/Login/img/Login/page_2 copy.png);
	border:1px solid rgb(26, 26, 26, 51);
}
</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>14</number>
        </property>
        <property name="rightMargin">
         <number>14</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnCloseButton">
          <property name="minimumSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#btnCloseButton{	
border-image: url(:/Login/img/Login/close_green_Normal.png);
	
}

QPushButton#btnCloseButton:hover{		
border-image: url(:/Login/img/Login/close_green_Click.png);
	
}


</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="page_3">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="bottomMargin">
           <number>90</number>
          </property>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>168</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labTips">
            <property name="styleSheet">
             <string notr="true">QLabel{
	font: 18px &quot;微软雅黑&quot;;
    color: #222222;
}
</string>
            </property>
            <property name="text">
             <string>更新检查</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>367</width>
              <height>98</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="labProgressBar">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>10</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>10</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_11">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>10</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>10</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_version">
                <property name="styleSheet">
                 <string notr="true">QLabel{
	font: 18px &quot;微软雅黑&quot;;
    color: #222222;
}

</string>
                </property>
                <property name="text">
                 <string>课堂有新版本可以更新</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>6</number>
                </property>
                <property name="topMargin">
                 <number>24</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_ver">
                  <property name="styleSheet">
                   <string notr="true">QLabel{
	font: 14px &quot;微软雅黑&quot;;
    color: #222222;
}

</string>
                  </property>
                  <property name="text">
                   <string>客户端v2.5.6.7</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>8</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>8</height>
                 </size>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QTextEdit" name="textEditMsg">
                  <property name="minimumSize">
                   <size>
                    <width>290</width>
                    <height>48</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>290</width>
                    <height>48</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">
QTextEdit{
background:rgb(245,245,245);
border:0px;
font: 12px &quot;微软雅黑&quot;;
color: rgb(85, 85, 85);
}
/*滚动条样式 开始*/
QScrollBar:vertical
{

width:6px;
background:rgb(245,245,245);
margin:0px,0px,0px,0px;

}

QScrollBar::handle:vertical
{
background-color: rgb(216, 216, 216);
border-radius:3px;
min-height:20;
border: 0px solid #0091f5;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical 
{

background:rgb(245,245,245);
border: 0px;
}</string>
                  </property>
                  <property name="lineWrapMode">
                   <enum>QTextEdit::WidgetWidth</enum>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                  <property name="html">
                   <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:12px; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14px;&quot;&gt;11111111111111手动阀打发士大夫手动阀手动阀手动阀撒旦飞洒发水电费撒的法大师傅撒范德萨大地方sadder为凤尾热舞发说法萨芬撒旦发生飞洒地方撒旦发射点发威锋网乳房&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QScrollBar" name="tabelScrollBar">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>48</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>48</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">/*滚动条样式 开始*/
QScrollBar:vertical
{

width:6px;
background:rgb(245,245,245);
margin:0px,0px,0px,0px;

}

QScrollBar::handle:vertical
{
background-color: rgb(216, 216, 216);
border-radius:3px;
min-height:20;
border: 0px solid #0091f5;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical 
{

background:rgb(245,245,245);
border: 0px;
}</string>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="horizontalSpacer_12">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>18</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>167</width>
                <height>29</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnUpdate">
              <property name="minimumSize">
               <size>
                <width>290</width>
                <height>40</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>290</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{	
font: 16px &quot;微软雅黑&quot;;
	color: rgb(255, 255, 255);
	border:0px;
	border-radius:4px;
	background-color: rgb(82, 204, 144);
}

QPushButton:hover{
	color: rgb(255, 255, 255);
	border:0px;
	border-radius:4px;
	background-color: rgb(52, 184, 119);
}

QPushButton:disable{
	color: rgb(255, 255, 255);
	border:0px;
	border-radius:4px;	
	background-color: rgb(99, 224, 162);
}

</string>
              </property>
              <property name="text">
               <string>立即升级</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>167</width>
                <height>29</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>167</width>
                <height>29</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnCancel">
              <property name="minimumSize">
               <size>
                <width>290</width>
                <height>42</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>290</width>
                <height>42</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton{	
font: 12px &quot;微软雅黑&quot;;
	color: rgb(136, 136, 136);
	border:0px;
	border-radius:4px;
	background-color: rgb(255, 255, 255);
}

QPushButton:hover{
	color: rgb(6, 6, 6);
	border:0px;
	border-radius:4px;
}

QPushButton:disable{
	color: rgb(255, 255, 255);
	border:0px;
	border-radius:4px;	

}

</string>
              </property>
              <property name="text">
               <string>暂不升级</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>167</width>
                <height>29</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page_2">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>512</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="leftMargin">
             <number>40</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	font: 14px &quot;微软雅黑&quot;;
    color: #222222;
}</string>
              </property>
              <property name="text">
               <string>下载中，已经完成</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labPerc">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QLabel{
	font: 14px &quot;微软雅黑&quot;;
    color: #222222;
}</string>
              </property>
              <property name="text">
               <string>45%</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="leftMargin">
             <number>40</number>
            </property>
            <property name="rightMargin">
             <number>40</number>
            </property>
            <property name="bottomMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QProgressBar" name="progressBar">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>3</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>3</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QProgressBar::chunk { background-color:rgb(82, 204, 144);;border-radius: 2px; }</string>
              </property>
              <property name="value">
               <number>24</number>
              </property>
              <property name="textVisible">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>60</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true">
font: 14px &quot;微软雅黑&quot;;
color:rgba(187,187,187,1);
</string>
            </property>
            <property name="text">
             <string>www.vhall.com</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../VLive.qrc"/>
 </resources>
 <connections/>
</ui>
